Alicerce/Sources/Logging/Log.swift

```diff
@@ -111,6 +117,12 @@ public final class Log {
 
         return (destination.minLevel.rawValue <= item.level.rawValue)
     }
+
+    private class func defaultLogItemWrittenHandler(destination: LogDestination, item: Item, error: Error?) {
+        if let error = error {
+            fatalError("💥: Failed to log item \(item) to destination \(destination)! Error: \(error)")
```

Comment:
It seems a bit drastic that any error in any destination will crash the app by default 😆 

However, as mentioned above I don't agree with this error propagation approach 😛